bigredbutton/frontend/src/app/layout/views/home/home.view.ts

```diff
@@ -1,7 +1,22 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { IRoomSettings, IMember } from '@app/lib/models';
 
 @Component({
   templateUrl: './home.view.html',
   styleUrls: ['./home.view.scss'],
 })
-export class HomeView {}
+export class HomeView implements OnInit {
+  public settings: IRoomSettings;
+  public members: IMember[];
+
+  constructor(
+    private readonly _route: ActivatedRoute,
+  ) {}
+
+  public ngOnInit() {
+    this.settings = this._route.snapshot.data.roomSettings;
```

Comment:
unsafe way. and maybe we move it into destructurization? const {snapshot: {data: {roomSettings, members}}} = this._route?